samples/ballerina-by-example/examples/http-cors/http-cors.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina.net.http;
+
+@Description {value:"Service level CORS headers applies globally"}
```

Comment:
Shall we say "Service level CORS headers applies globally for each resource" ?

Reply:
Ack